### ISSUE TITLE
Enable login and entry-administration in the frontend

### ISSUE THREAD

**Issue by wellemut** [kartevonmorgen/kartevonmorgen.ts#152]
## Goal:
Implement whole administration of entries in Frontend, so that we can get rid of the backend html. like https://openfairdb.org/entries/9ba1a9e4138c4d2dbaa776fead85d098
![grafik](https://user-images.githubusercontent.com/15019030/210576386-e387f543-08e5-4265-bdd8-3b6bb7c0831b.png)

## Implementation

- [ ] Setup login-formular on https://www.kartevonmorgen.org/login (equal to https://openfairdb.org/login)

![grafik](https://user-images.githubusercontent.com/15019030/210577329-86e6fa70-0be9-4f88-84a0-18413322fbec.png)
Use login-API to get user cockie: https://app.swaggerhub.com/apis/Kartevonmorgen/openfairdb/0.10.5#/Users/post_login
- [ ] Setup register-formular on https://www.kartevonmorgen.org/register (equal to https://openfairdb.org/register)
![grafik](https://user-images.githubusercontent.com/15019030/210577721-45270f05-a639-4a03-a7b8-34e7182ab8a3.png)

- [ ] Add Button to History and to Archive/reject operations in the footer of the entry. **only for logged-in user**
Button-Names
  - Link from the Version number and "last edit ..." to the Entry History page and add the link hint "View Entry History"
  - Use Button "Delete this entry" Instead of "View history or archive this entity" (keep the yellow warning for login)
![grafik](https://user-images.githubusercontent.com/15019030/210580663-89cc0e50-b9d5-42e6-8733-dd373948c4e0.png)

![grafik](https://user-images.githubusercontent.com/15019030/210578748-27e0d36f-5752-4095-acbd-35e24975e9e2.png)

- [ ] If users are not logged in, show the "login-button" in the footer of each entry instead at the button right corner of the footer.
  - No link to the History

- [ ] Add History view of the entry - open in new tab!
Similar to https://openfairdb.org/places/de6ac9a92bde46d0b34bd5961196ec12/history
See API: https://app.swaggerhub.com/apis/Kartevonmorgen/openfairdb/0.10.5#/Entries%2FPlaces/get_places__id__history__revision_
![grafik](https://user-images.githubusercontent.com/15019030/210579255-ff8685b9-0628-40d6-b06b-f62bcb8da49e.png)
  - [ ] This "History-List" needs the current Title of the Entry as headline which is linkt to the entry on the map.

- [ ] Add the Delete-Page for the entry
  - [ ] The comment-field with the three buttons "reject" |  "archive" |  "confirm"  can be implemented as a popup. 
![grafik](https://user-images.githubusercontent.com/15019030/210579720-123a9a18-cf8d-4409-940d-01d8a319e978.png)
- there is no need of the change-button, if the three options are three buttons and not a dropdown!
- API for review: https://app.swaggerhub.com/apis/Kartevonmorgen/openfairdb/0.10.5#/Entries%2FPlaces/post_places__ids__review

- [ ] Redirect Login/register Button in the Hamburger Menu  to the frondend (currently backend)
![grafik](https://user-images.githubusercontent.com/15019030/210578219-3baa0239-0186-408f-aaf4-01194ffc91ae.png)

(still not needed on landing-page (https://github.com/kartevonmorgen/kartevonmorgen.ts/issues/118)

**Comment by wellemut** [comment]
@navid-kalaei today in a call with @flosse and @alex0107 we decided, that it would be the best, if all administrational tasks can be done in your frontend.

Do you mind to implement it after the other issues?

**Comment by flosse** [comment]
> @navid-kalaei today in a call with @flosse and @alex0107 we decided, that it would be the best, if all administrational tasks can be done in your frontend.

not really. The only thing that has to be implemented by @navid-kalaei are the functionalities that are used by a user, such as register, login, subscribe, unsubscribe etc.
All other administrative tasks may or may not be performed by the main frontend. It also could be a complete separate implementation.
